tests/cloner_test.go

```diff
@@ -120,3 +124,19 @@ func doCloneTest(f *framework.Framework, targetNs *v1.Namespace) {
 		Expect(err).ToNot(HaveOccurred())
 	}
 }
+
+func cloneOfAnnoExistenceTest(f *framework.Framework, targetNamespaceName string) {
+	// Create targetPvc
+	By("Creating target pvc")
+	_, err := utils.CreatePVCFromDefinition(f.K8sClient, targetNamespaceName, utils.NewPVCDefinition(
+		"target-pvc",
+		"1G",
+		map[string]string{controller.AnnCloneRequest: f.Namespace.Name + "/" + sourcePVCName, controller.AnnCloneOf: "true"},
+		nil))
+	Expect(err).ToNot(HaveOccurred())
+	By("Checking no cloning pods were created")
+	_, err = f.FindPodByPrefix(common.ClonerSourcePodName)
```

Comment:
Sorry, I know this isn't you code but there's a slight idiomatic change I'd make to _FindPodByPrefix_:
```
func FindPodByPrefix(clientSet *kubernetes.Clientset, namespace, prefix, labelSelector string) (*k8sv1.Pod, error) {
...
	err := wait.PollImmediate(2*time.Second, PodCreateTime, func() (bool, error) {
		podList, err := clientSet.CoreV1().Pods(namespace).List(metav1.ListOptions{
			LabelSelector: labelSelector,
		})
		if err == nil {
			for _, pod := range podList.Items {
				...
			}
		}
		return found, nil  // <==== here, don't need 2 returns, remove return in `err == nil` block
	})
```

Reply:
sure.